src/Components/RosTable/redux/systemsTableReducer.js

```diff
@@ -0,0 +1,35 @@
+const initialState = {
+    loading: false,
+    systemsData: [],
+    systemError: {}
+};
+
+const systemsTableReducer = () => {
+    const tableReducer = (state = initialState, action) => {
+        switch (action.type) {
```

Comment:
I'm not a huge fan of switches as they tend to be too wordy and not so obvious, you can use `applyReducerHash` and have an object mapper

```JS
return applyReducerHash({
  'FETCH_CLOUD_SYSTEMS_LIST_PENDING': (state) => ({ ...state, loading: true }),
  'FETCH_CLOUD_SYSTEMS_LIST_FULFILLED': (state, { payload }) => ({ ...state, loading: false, systemsData: payload }),
  'FETCH_CLOUD_SYSTEMS_LIST_REJECTED': (state, { payload }) => ({ ...state, loading: false, systemError: payload }),
}, initialState)
```